data/autoyast_qam/15_installation.xml.ep

```diff
@@ -99,15 +102,21 @@
       <pattern><%= $pattern %></pattern>
       % }
     </patterns>
-  <services-manager>
-    <services>
-      <disable config:type="list"/>
-      <enable config:type="list">
-        <service>sshd</service>
-      </enable>
-    </services>
-  </services-manager>
+    <services-manager>
```

Comment:
looks like just code formatting , right ? 
because we don't have any confirmed standard for xml can you please revert this part ? Someone else editor might change it back and later it would be really hard to use git blame for this lines

Reply:
Intentional indentation change as those few lines were not nested correctly (some else editor should change all or nothing; not only a part of the file). By the way, it is not really a xml file :smile: